bin/cam.js

```diff
@@ -1,14 +1,15 @@
-const cam = require('../lib/camera')
+// const scanner = require('../lib/scanner')
+const scanner = require('../lib/mocks/scanner')
```

Comment:
Why is using the mock scanner here?

Reply:
I just need to call the `scanPhotoCard` method which is the same between mocked and un-mocked version.
Unfortunately I cannot use the un-mocked version on my dev environment as I'm missing v4l2camera and it would raise an exception